virttest/tests/unattended_install.py

```diff
@@ -1296,6 +1296,8 @@ def copy_images():
         error_context.context(msg, logging.info)
         download.get_file(nfs_link, local_link)
         params[media] = local_link
+    # at this stage we need vm specific parameters
+    params = params.object_params(vm.name)
```

Comment:
Looks it is needed in the earlier stage, before line 1262, since almost all the params are per VM ones.

Reply:
Great, I only put it at a line where I was sure it will be needed. If it is needed earlier than I will move it right up to line 1262.

Reply:
I had to also move the code `vm = env.get_vm(params["main_vm"])` from line 1278 since it is needed in order to restrict the parameters.

Reply:
Yes, I think so, or you can have something like `params.object_params(params["main_vm"])` at there, thanks.

Reply:
And btw, it is handled well by the wrapper in tp-qemu for multiple VMs installation cases, but unfortunately the single VM one didn't take the benefit at that time.
https://github.com/autotest/tp-qemu/blob/master/generic/tests/unattended_install.py#L38

Reply:
> Yes, I think so, or you can have something like `params.object_params(params["main_vm"])` at there, thanks.

But I think what I suggested with the last push is a bit more natural. It is part of the usual workflow of obtaining a vm object and its parameters.

Reply:
> And btw, it is handled well by the wrapper in tp-qemu for multiple VMs installation cases, but unfortunately the single VM one didn't take the benefit at that time.
> https://github.com/autotest/tp-qemu/blob/master/generic/tests/unattended_install.py#L38

Indeed, but I guess the current change is a minimal one to achieve the same effect here :)

Reply:
> > Yes, I think so, or you can have something like `params.object_params(params["main_vm"])` at there, thanks.
> 
> But I think what I suggested with the last push is a bit more natural. It is part of the usual workflow of obtaining a vm object and its parameters.

ok, but looks you forgot to bring line 1278 to there, would you please check it?

Reply:
turns out the hunk was staged but not committed ^^ done now